x-pack/legacy/plugins/integrations_manager/server/config.schema.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { schema, TypeOf } from '@kbn/config-schema';
+
+export const config = {
+  schema: schema.object({
+    enabled: schema.boolean(),
```

Comment:
Related to the above question about `joi`. These two schemas seem _almost_ the same but not quite. Are they meant to be independent or identical? If identical, can we move the definition to `common` and have each file `import` them?

Reply:
They should be equivalent, I'll look into it.

Reply:
fwiw, the Joi one should completely disappear as soon as the shim disappears, so this may depend on whether you want that abstraction to stick around after that happens and it's only being used in the one place. (But for now I'm not sure if they can really be completely shared since one is raw Joi and the other is config-schema)